#
# Copyright 2002 Sun Microsystems, Inc. All rights reserved.
# SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
#
# Log messages used by the javax.enterprise.system.activation logger 
# LogDomains.ACTIVATION_LOGGER, subsystem activation.
# See java.text.MessageFormat for details on constructing messages.
# Messages in this file must be of the form 
# <Subsystem><4CharIntegerMessageId>:<message text>
#
#example.message1=TLS0001:Test Message. Parameter 1={0} parameter 2={1}
#example.message2=TLS0002:Date Message. Parameter 1={0,date} Parameter 2={1,time}
#example.message3=TLS0003:Number Message. Parameter 1={0,number,integer}
synchronization.skipping_synchronization=SYNC001: Unable to communicate with Domain Administration Server. Skipping synchronization.
synchronization.no_connection=SYNC002: Unable to connect to admin server: {0}.
synchronization.no_matching_server=SYNC076: Domain Administration Server failed to recognize server instance or node agent by the name {0}.
synchronization.no_matching_agent=SYNC077: Connection to DAS succeeded. However DAS failed to recognize this node agent {0}. This could be because this node agent's config was deleted from the DAS. Or it could be because a DAS different from the one with which this Node Agent bounded is now running at the host = {1} and port = {2}. Please perform any of the following three options. 1. Add the node agent config explicitly 2. Delete this node agent 3. Start the correct DAS on the host and port this node agent was created with.
SYNC002.diag.cause.1=User name, password, host name or port number may be incorrect.
SYNC002.diag.cause.2=DAS may be down or unreachable.
SYNC002.diag.check.1=Check if User name, password, host name and port number are correct.
SYNC002.diag.check.2=Make sure DAS is running and can connect from the current instance(host).
synchronization.req_mod_file_info=SYNC003: Request for {0} is modified.
synchronization.req_not_mod_file_info=SYNC004: Request for {0} is not modified.
synchronization.synchronization_start_time=SYNC005: Received synchronization request at time [{0}] from [{1}].
synchronization.malformed_xml=SYNC006: MalformedURL: {0} and exception message is {1}.
SYNC006.diag.cause.1=Synchronization module subsystem's meta data xml file is corrupted.
SYNC006.diag.check.1=Please check synchronization-meta-data.xml and na-synchronization-meta-data.xml exist in appserv-se.jar and is a valid xml file.
synchronization.config.io_exception=SYNC007: IOException: {0} and exception message is {1}.
SYNC007.diag.check.1=Check if synchronization-meta-data.xml is not part of server classpath.
synchronization.parser_config_exception=SYNC008: ParserConfiguationException: {0}.
SYNC008.diag.check.1=synchronization-meta-data.xml is not valid. 
synchronization.sax_parse_exception=SYNC009: Parsing error, line {0,number,integer}, uri {1} and message is {2}.
SYNC009.diag.check.1=synchronization-meta-data.xml found in server classpath is not well formed. 
synchronization.time_taken=SYNC010: Synchronization completed in {0} seconds.
synchronization.util.concurrent.barrier_tripped=SYNC011: Synchronization barrier tripped: {0}
synchronization.request_start=SYNC012: Started synchronizing: {0}
synchronization.process_command=SYNC013: Processing response for synchronization request file: {0}
synchronization.timestamp_command_error=SYNC014: Unable to update synchronization timestamp.
synchronization.store_commit_error=SYNC015: Unable to commit synchronization changes.
synchronization.store_rollback_error=SYNC016: Unable to rollback synchronization changes.
synchronization.api.conn_reg_get_failed=SYNC017: Getting a mbean server connection for instance {0} from registry failed.
synchronization.api.jmx_ctor_failed=SYNC018: JMXFileTrasfer constructor for server {0} failed with IOException.
synchronization.api.jmx_config_ctor_failed=SYNC019: JMXFileTrasfer constructor for server {0} failed with ConfigException.
synchronization.api.connect_error=SYNC020: Connection is no longer available, please reconnect.
synchronization.api.create_local_file_failed=SYNC021: Could not create the local file {0}.
synchronization.api.transfer_exception=SYNC022: During the file transfer of {0}, there was a failure caused by IOException.
synchronization.api.no_absolute=SYNC023: {0} is an absolute path, it is not allowed.
synchronization.config_no_appclient=SYNC024: Can not determine Web modules configured for server: {0}
synchronization.config_no_connector=SYNC025: Can not determine Connector modules configured for server: {0}
synchronization.config_no_lifecycle=SYNC026: Can not determine Lifecycle modules configured for server: {0}
synchronization.sending_file=SYNC027: Sending synchronization file of size: {0} byte(s), checksum {1}.
synchronization.timestamp_exception=SYNC028: Timestamp file {0} could not be opened. Exception message is {1}.
synchronization.sync_fail=SYNC029: There was an exception during synchronization. Please try again and a full synchronization will be initiated. If the problem persists for a server instance, you may backup and remove the cache repository (for example, directories under nodeagents/na1/server1/) and restart the node-agent (ex. asadmin stop-node-agent; start-node-agent --startinstances=false) and the server instance (ex. asadmin start-instance). All necessary configuration for the server instance (server1 in example) will be downloaded from Domain Administration Server by synchronization. 
SYNC029.diag.cause.1=This could be either a config Exception or IOException
SYNC029.diag.check.1=Please check domain.xml and das.properties. If the problem persists, you may backup and remove applications, generated, config, docroot and lib directories in the local repository and re-start the server. The local repository will be synchronized again. 
synchronization.config_not_found=SYNC030: Can not get config information for server: {0}
synchronization.ts_not_created=SYNC031: Timestamp file {0} creation failed.
synchronization.ts_exists=SYNC032: Timestamp file {0} already exists.
synchronization.start_request=SYNC033: Starting synchronization request for {0}.
synchronization.done_request=SYNC034: Synchronization completed for request {0}.
synchronization.remove_olddir=SYNC035: Removing old directory {0}.
synchronization.rename_dir=SYNC036: Renaming from {0} to {1}.
synchronization.merge_dirs=SYNC037: Merging files/dirs from {0} to {1}.
synchronization.get_backup=SYNC038: Get backup file for {0}.
synchronization.save_momento=SYNC039: Saving momento for {0}.
synchronization.rollback_momento=SYNC040: Rollbacking momento for {0}.
synchronization.commit_momento=SYNC041: Committing momento for {0}.
synchronization.no_backup_dir=SYNC042: No backup directory {0}, nothing done at commit time of momento.
synchronization.no_sync_file=SYNC043: File/Directory {0} was not modified, so not synchronized.
synchronization.ts_created=SYNC044: Timestamp file {0} created.
synchronization.file_received=SYNC045: Received request file/directory: {0}
synchronization.vote_commit=SYNC046: Could complete synchronization transaction branch for: {0}. If all other transaction branches passed, synchronization will be succeded.
synchronization.vote_rollback=SYNC047: Could not complete synchronization transaction for: {0}.
SYNC047.diag.cause.1=Connection problem during download.
SYNC047.diag.cause.2=File(s) could not be written to synchronization store.
SYNC047.diag.check.1=Check if DAS is up. Check for connection errors during download.
SYNC047.diag.check.2=Check if synchronization repository has enough space and has permission. If the repository is on NFS, may be NFS is down.
synchronization.tran_commit=SYNC048: All synchronization requests succedded for: {0}
synchronization.tran_fail=SYNC049: Transaction is committed, but changes to {0} could not be written to disk.
SYNC049.diag.cause.1=File(s) could not be written to synchronization store.
SYNC049.diag.check.1=Check if synchronization repository has enough space and has permission. If the repository is on NFS, may be NFS is down.
synchronization.check_sum=SYNC050: Checksum of synchronizated file: {0} 
synchronization.file_received=SYNC051: Received request file/directory: {0}
synchronization.config_no_apps=SYNC052: Can not determine Applications configured for server: {0}
SYNC052.diag.cause.1=domain.xml is incorrect.
SYNC052.diag.check.1=Make sure domain.xml errors are fixed. It may have been manually editied.
synchronization.config_no_ejbs=SYNC053: Can not determine Ejb modules configured for server: {0}
synchronization.config_no_web=SYNC054: Can not determine Web modules configured for server: {0}
synchronization.cleaner.file_remove_error=SYNC055: Unexpected error while removing file: {0}
synchronization.cleaner.file_removed=SYNC056: Removed old file: {0}
synchronization.cleaner.servername_error=SYNC057: com.sun.aas.instanceName is not defined in the system property.
synchronization.cleaner.configctx_error=SYNC058: Unable to create config contextin repository cleaner.
synchronization.cleaner.begin=SYNC059: Starting application server repository cleaner.
synchronization.cleaner.end=SYNC060: Application server repository cleanup complete.
synchronization.start.download.bits=SYNC061: Starting synchronization for {0}. 
synchronization.end.download.bits=SYNC062: Synchronization for {0} is complete. Total time spent {1} milli second(s).
synchronization.done.keyfile.connect=SYNC063: Connected to DAS [{0}] successfully.
synchronization.done.keyfile.getloc=SYNC064: Gettting the location of keyfile for server instance - {0}.
synchronization.done.keyfile.unresolve=SYNC065: Un-resolved location path - {0} to {1}.
synchronization.done.keyfile.disconnect=SYNC066: Receive keyfile {0} from DAS. 
synchronization.done.keyfile.download=SYNC067: Finished keyfile synchronization of {0}. Close connection to DAS.
synchronization.prevote.failed=SYNC068: Prevote failed with exception message {0}.
synchronization.remove.failed=SYNC069: Server was unable to remove [{0}] for unknown reason. This file is not necessary for sever operation and can be removed.
synchronization.mbean.failed=SYNC070: Unexpected error while processing synchronization request.
synchronization.zipfile.notfound=SYNC071: Invalid zip location [{0}]. File does not exist.
failureOnReadingSecurityIdentity=SYNC072: Reading identity from system id failed
synchronization.cleaner.time_taken=SYNC073: Repository cleaning cycle took {0} seconds to complete.
synchronization.audit_fail=SYNC074: There was an exception during synchronization audit.
SYNC074.diag.cause.1=This could be a network communication error.
SYNC074.diag.check.1=Please check domain.xml, das.properties and server log.
synchronization.audit_report_success=SYNC075: Synchronization audit completed successfully. All the necessary {0} files exist in the local repostiroy. There were no extra files found in the audit.
synchronization.retry_synchronization=SYNC075: Initial attempt to synchronize failed. Re-trying.
SYNC075.diag.cause.1=This could be a network communication error or Domain Administration Server is not running.
SYNC075.diag.check.1=Please ensure that Domain Administration Server is running. If DAS is accessible and the problem persists, you may backup and remove applications, generated, config, docroot and lib directories in the local repository and re-start the server. The local repository will be synchronized again. 
.
